pallets/dapp-staking-v3/src/types.rs

```diff
@@ -1477,24 +1477,16 @@ pub enum TierThreshold {
         amount: Balance,
         minimum_amount: Balance,
     },
-}
-
-impl TierThreshold {
-    /// Used to check if stake amount satisfies the threshold or not.
-    pub fn is_satisfied(&self, stake: Balance) -> bool {
-        match self {
-            Self::FixedTvlAmount { amount } => stake >= *amount,
-            Self::DynamicTvlAmount { amount, .. } => stake >= *amount,
-        }
-    }
-
-    /// Return threshold for the tier.
-    pub fn threshold(&self) -> Balance {
-        match self {
-            Self::FixedTvlAmount { amount } => *amount,
-            Self::DynamicTvlAmount { amount, .. } => *amount,
-        }
-    }
+    /// Entry into the tier is mandated by a fixed percentage of the total issuance as staked funds.
```

Comment:
You can freely remove the above values, they're not needed anymore.

Reply:
I kept them for mock tests on local chains, but if they're not used in production/testnet in the future, I'll get rid of these variants. Some adjustments may be needed to make the tests more agnostic to changes in the total issuance, I'll refactor them later.

Reply:
For the local chain - if we can run astar-dev, shibuya-dev, shiden-dev with percetange based value, it shouldn't be a problem for the local chain either then.

For mock/tests, we shouldn't include extra values into enum/structs/etc.
Or at least they should be gated under the test feature, IMO.

Reply:
Sure, I mixed both but I was thinking of mock/tests 😁
I also agree, so I removed the extra values in the enum and made sure all the tests passed.

Genesis config were updated for local chains too to use the same percentage-based values.